src/main/java/robotbuilder/actions/ExporterAction.java

```diff
@@ -33,13 +33,15 @@ public ExporterAction(String description) {
     @Override
     public void actionPerformed(ActionEvent ae) {
         boolean newProject = false;
-        MainFrame.getInstance().setCursor(Cursor.WAIT_CURSOR);
+        MainFrame.getInstance().setCursor(new Cursor(Cursor.WAIT_CURSOR));
```

Comment:
Since Cursor.WAIT_CURSOR is static, this shouldn't be necessar

Reply:
I'm making the change but just fyi it warn that the version you are reccomending is depreciated which is why i changed it

Reply:
Hmm, I don't see that in the JDK docs. However, if it's to fix a warning then go ahead and keep it.

Reply:
https://docs.oracle.com/javase/8/docs/api/java/awt/Frame.html#setCursor-int-
here is the deprecation thing i found